### PR TITLE
Explained about the HTML Links.

### LINKED ISSUE #1292: Frontend : HTML Links
<hr>

## Description 📜

Explain about the HTML links.
<hr>

## Domain of Contribution 📊

- [x] Frontend Dev (HTML, CSS, JS)

<hr>

### PR DESCRIPTION
**Pull request by Jaya3112**
<hr>

## Description 📜

Explained about the HTML Links. 

Fixes #1292 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev HTML/CSS/JS

<hr>

## Checklist ✅

- [x ] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [ x] I have performed a self-review of my own code or work.
- [ x] I have commented my code, particularly in hard-to-understand areas.
- [x ] My changes generates no new warnings.
- [x ] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/FrontEnd/HTML Links.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<h1 align="center">HTML Links</h1>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:1]: Remove the HTML from this. Make it markdown.

File: Web_Development/FrontEnd/HTML Links.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    • A Html link exhibit in different colors on the basis of whether it is visited , unvisited or active.
+
+        (a) for visited : links should be underlined and blue.
+
+        (b) for Unvisited : link should be underlined and purple.
+
+        (c) for active : link should be underlined and red.
+
+    • We can also create a bookmarks using HTML links.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:27]: Use Markdown format, not text format.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please convert your document into Markdown not in Text / HTML.

File: Web_Development/FrontEnd/HTML Links.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+## HTML Links
+
+    • To link an external documents to the current document we use <link> HTML element. It is also denoted with hyperlinks. It can be written
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:3]: Make it paragraph.

File: Web_Development/FrontEnd/HTML Links.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+## HTML Links
+
+    • To link an external documents to the current document we use <link> HTML element. It is also denoted with hyperlinks. It can be written    
+      inside the <body> or inside the <head>.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:4]: Use it in single line. Make `<body>` as inline code like this:

```suggestion
      inside the `<body>` or inside the `<head>`.
```

File: Web_Development/FrontEnd/HTML Links.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+            example :
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:6]: No spaces on the front please.

File: Web_Development/FrontEnd/HTML Links.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+            <link rel="stylesheet" href="">
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:7]: Use a code fence.

File: Web_Development/FrontEnd/HTML Links.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+    • "href" attribute is used to provide the path or way to the stylesheet(mostly it is URL).
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:9]: Use correct way of using unordered list in Markdown.
Use inline code for "href": Use `href` not "href".

File: Web_Development/FrontEnd/HTML Links.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:10]: No need of space between LIs.

File: Web_Development/FrontEnd/HTML Links.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+    • "rel" attribute represents the relationship and imply one of the main features of the <link> HTML element.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:11]: Use correct way of using unordered list in Markdown.
Use inline code for "href": Use `href` not "href".

File: Web_Development/FrontEnd/HTML Links.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+    • It is not neccesary that links should be in text type. It can be of image , button and other HTML elements.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:13]: Use correct way of using unordered list in Markdown.

File: Web_Development/FrontEnd/HTML Links.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+
+    • Anchor tag <a> is the most common tag which is used to create the hyperlink in an interesting way.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:15]: Use correct way of using unordered list in Markdown.
Use inline code for "href": Use `href` not "href".

File: Web_Development/FrontEnd/HTML Links.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+        example:
+           <a href="https://gwoc.girlscript.tech/">GIRLSCRIPT</a>
+
+    • A Html link exhibit in different colors on the basis of whether it is visited , unvisited or active.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:19]: Use correct way of using unordered list in Markdown.
Use inline code for "href": Use `href` not "href".

File: Web_Development/FrontEnd/HTML Links.md [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+
+        (a) for visited : links should be underlined and blue.
+
+        (b) for Unvisited : link should be underlined and purple.
+
+        (c) for active : link should be underlined and red.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:25]: Use correct way of using ordered list in Markdown.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please review the comments.

File: Web_Development/FrontEnd/HTML Links.md [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+**Example**
+
+```html
+<link rel="stylesheet" href="" />
+```
+
+- `href` attribute is used to provide the path or way to the stylesheet(mostly it is URL) 
+- `rel` attribute represents the relationship and imply one of the main features of the `<link>` HTML element.
+
+It is not neccesary that links should be in text type. It can be of image, button and other HTML elements.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:16]: This is completely wrong. Please check and come back.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Contents are completely wrong. Please refer to [`<a>`: The Anchor element - HTML: HyperText Markup Language | MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a).

File: Web_Development/FrontEnd/HTML Links.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+<a href="https://gwoc.girlscript.tech/">GIRLSCRIPT</a>
+```
+
+A HTML link exhibit in different colors on the basis of whether it is visited, unvisited or active.
+
+1. For Visited: links might be underlined and blue. 
+2. For Unvisited: link might be underlined and purple.
+3. For Active: link might be underlined and red.
+
+We can also create a bookmarks using HTML links.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:31]: Can you add some examples for this? Tell more about Internal vs. External Links and the use of `target` attribute.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: Web_Development/FrontEnd/HTML Links.md (hunk 15 above)
Comment (Jaya3112) [reply Web_Development/FrontEnd/HTML Links.md:31]: sure

File: Web_Development/FrontEnd/HTML Links.md [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+```
+
+## Internal Links
+
+An internal link is used in html page to navigate in a proper way in the webpage.If we want to go to any particular destination in the same page by clicking on a link then we create an internal link in the webpage . For this we use `<a>` tag to create a new link. 
+
+**Example**
+
+```html
+<a href= “#link”> FirstLink</a>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:68]: The quotes are wrong.

File: Web_Development/FrontEnd/HTML Links.md [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+```
+
+## External Links
+
+An external link is used to make interconnection between two html webpages. When we want to direct to some other page or any other URL by clicking on a link on webpage then we create an external link. For this we again use `<a>` tag in html webpage
+
+**Example**
+
+```html
+<a href= “name or address of webpage”> text</a>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:78]: The quotes are wrong.

File: Web_Development/FrontEnd/HTML Links.md (hunk 16 above)
Comment (praveenscience) [reply Web_Development/FrontEnd/HTML Links.md:68]: Please don't use space before quotes.

File: Web_Development/FrontEnd/HTML Links.md (hunk 17 above)
Comment (praveenscience) [reply Web_Development/FrontEnd/HTML Links.md:78]: Please don't use space before quotes.

File: Web_Development/FrontEnd/HTML Links.md [hunk 18]
```diff
@@ -0,0 +1,87 @@
...
+``` 
+ 
+ 
+ 
+ 
+ 
+ 
+ 
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML Links.md:87]: Remove empty lines.

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Review by praveenscience — APPROVED** [review]
Looks good to me. 👍

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Nice content. Also try adding topics like an image as a link, button as a link, and link titles.

**Comment by kashishahuja2002** [comment]
Please merge this PR, @praveenscience

**Review by praveenscience — APPROVED** [review]
👍

**Pull request merged**